resources/ticket.scss

```diff
@@ -0,0 +1,217 @@
+@import "vars";
+
+form#ticket-form {
+    display: block;
+    margin: 0 auto;
+    width: 100%;
+    max-width: 750px;
+    padding-top: 1em;
+
+    #id_title {
+        width: 100%;
+    }
+
+    .submit {
+        margin: 10px 0 0 auto;
+    }
+}
+
+#ticket-list {
+    .fa-check-circle-o {
+        color: #44AD41;
```

Comment:
Maybe we should have a var in this file for this and the other color, since they're used twice.

Reply:
I don't feel strongly about this.